README.md

```diff
@@ -27,3 +27,37 @@ If desired timestamps can be prefixed to the debug message. Timestamp output can
 
 # How-To-Use Advanced
 Normally all debug output is redirected to the primary serial output of each board (`Serial`). In case you want to redirect the output to another output stream you can make use of `setDebugOutputStream(&Serial2)`.
+
+# Documentation
+### 1. Debug :
+Arduino_DebugUtils Object that will be used for calling member functions.
+
+### 2. Debug.setDebugLevel(int const debug_level) :
+Parameter debug_level in order of lowest to highest priority are : `DBG_NONE`, `DBG_ERROR`, `DBG_WARNING`, `DBG_INFO` (default), `DBG_DEBUG`, and `DBG_VERBOSE`. 
+Return type: void.
+Example:
```

Comment:
```suggestion

Return type: void.

Example:
```
Unfortunately, Markdown doesn't render single line breaks. This caused the original rendered output to look like:
>... , and `DBG_VERBOSE`. Return type: void. Example:

There are several ways to cause a line break in the rendered output, but I think the extra vertical whitespace caused by the empty lines I added makes things more readable.